cookbooks/vagrant/recipes/default.rb

```diff
@@ -240,30 +239,6 @@
 EOH
 end
 
-if RUN_TESTS then
-  execute "setup ckan test db" do
-    user "postgres"
-    not_if "sudo -u postgres psql -c '\\l' | grep ckan_test"
-    command "createdb -O ckan_default ckan_test -E utf-8"
-  end
-  
-  execute "setup datastore test db" do
-    user "postgres"
-    not_if "sudo -u postgres psql -c '\\l' | grep datastore_test"
-    command "createdb -O ckan_default datastore_test -E utf-8"
-  end
-  
-  bash "reset/populate the test db" do
-    not_if "psql -c '\\dt' ckan_test | grep resource_revision", :user => "postgres"
-    user USER
-    cwd CKAN_DIR
-    code <<-EOH
-  source #{HOME}/pyenv/bin/activate
-  nosetests --ckan --reset-db --with-pylons=test-core.ini ckan
-  EOH
-  end
-end
-
```

Comment:
This part is already conditional, I think we can leave it like that. The default should set to `RUN_TESTS = false`. There is no need to remove it.